app/models/events/new_comment.rb

```diff
@@ -1,22 +1,10 @@
 class Events::NewComment < Event
-  def self.publish!(comment)
-    event = create!(kind: 'new_comment',
-                    eventable: comment,
-                    discussion: comment.discussion,
-                    created_at: comment.created_at)
-
-    Events::CommentRepliedTo.publish! comment if comment.is_reply?
-
-    comment.mentioned_group_members.
-            without(comment.parent_author).find_each do |mentioned_user|
-      Events::UserMentioned.publish!(comment, mentioned_user)
-    end
 
-    BaseMailer.send_bulk_mail(to: UsersToEmailQuery.new_comment(comment)) do |user|
-      ThreadMailer.delay.new_comment(user, event)
-    end
-
-    event
+  def self.publish!(comment)
```

Comment:
This is getting pretty darn close to being able to generalize the `publish!` method and more it up into event.

Something like

```
# event.rb
def self.publish!(eventable)
  create(kind: kind, eventable: eventable, created_at: eventable.created_at).tap do |event|
    EventBus.broadcast("#{kind}_event", event
  end
end
```